docker-compose.yml

```diff
@@ -17,6 +17,21 @@ services:
       MYSQL_PASSWORD: starchart_password
       MYSQL_ROOT_PASSWORD: root_password
 
+  test_database:
```

Comment:
I don't love that we're going to have to run 2 instances of MySQL in development.  Why not run 2 databases in the same instance?

See https://stackoverflow.com/questions/39204142/docker-compose-with-multiple-databases.

Reply:
Make sense. Let me look into it.